tutorial.uvfits

```diff
@@ -0,0 +1,91 @@
                                       OBJECT  = 'zenith  '                                                            TELESCOP= 'PAPER   '                                                            LAT     =    -30.7215277777779                                                  LON     =    21.42830555555556                                                  ALT     =    1073.000000008382                                                  INSTRUME= 'PAPER   '                                                            EPOCH   =               2000.0                                                  PHSFRAME= 'icrs    '                                                            HISTORY MDLVIS: created file.                                                   HISTORY MDLVIS: /opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin/HISTORY mdlvis.py -C psa898_v003 -a all -s pic,for,her,hyd,cen --nchan=128 --sfrHISTORY eq=.1 --sdf=0.00078125 --inttime=2 --startjd=2456200.5 --endjd=2456200.5HISTORY 0069 --pol=xx                                                           HISTORY PULL ANTPOLS:/Users/jpober/src/capo/scripts/pull_antpols.py new.uv -a 0_HISTORY 1                                                                       HISTORY PUTHD: Miriad PutHd: Version 1.0 13-dec-96                              HISTORY PUTHD: Executed on: 2016-06-01T16:18:51.0                               HISTORY PUTHD: Command line inputs follow:                                      HISTORY PUTHD:   in=new.uvA/telescop                                            HISTORY PUTHD:   value=PAPER                                                    HISTORY PUTHD:   type=ascii                                                     HISTORY   Read/written with pyuvdata version: 1.4.2.  Git origin: https://githubHISTORY .com/RadioAstronomySoftwareGroup/pyuvdata.git.  Git hash: 4feb01c2d61b3bHISTORY bd2bb5a475763b829ade7ece8d.  Git branch: tutorial_convert_to_gain.  Git HISTORY description: v1.4-557-g4feb01c2-dirty.                                  END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             �
```

Comment:
This file should not have been added to the PR, it should be deleted when the tutorial completes.

Reply:
Yikes! Let me try to fix that...

Reply:
Let me know if you need help. You may have gotten it in by using something like `git add -a` which many people use but I find very dangerous because it just adds all changes without making you think through which ones you actually want.

Reply:
I'm relatively sure I got rid of the errant tutorial file, but the branch still isn't passing all the tests. What's going on? Sorry that I'm such a github neophyte!

Reply:
Oh, I think I see what's going on: my documentation is actually bad! I reference 'caltype' when it should be 'cal_type'. 

Actually, now that I look, I'm seeing contradictions in UVCal.py -- it looks like cal_type has been changed to _cal_type, and wants to be an internal hidden variable now. Should I update the methods set_gain(), set_delay(), set_unknown(), and so on that reference UVCal.cal_type and change it to UVCal._cal_type? Should there be another method that just reports a UVCal object's _cal_type, now that it's a hidden variable?

I guess first I'll change the tutorial so that doctest will run without failing.

Reply:
So the `_cal_type` points to the UVParameter object, the `cal_type` is the property that behaves to the user like an attribute and it just points to the `value` attribute on the `_cal_type` object. We expect that most users will just use the properties (the ones that don't start with an underscore), but the UVParameter objects contain essential information that support the various checks that we run.